packages/devtools_app/lib/src/screens/memory/primitives/ui.dart

```diff
@@ -0,0 +1,6 @@
+// Copyright 2022 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Width of application when memory buttons lose their text.
```

Comment:
"Width of application"? What does that mean? Do you mean width of the screen?

Reply:
added 'window'